tests/brevitas/core/test_float_quant.py

```diff
@@ -233,3 +235,35 @@ def test_inner_scale(inp, minifloat_format, scale):
             out_nans = out.isnan()
             expected_out_nans = expected_out.isnan()
             assert torch.equal(out[~out_nans], expected_out[~expected_out_nans])
+
+
+@given(
+    minifloat_format_and_value=random_minifloat_format_and_value(
+        min_bit_width=4, max_bit_with=10, rand_exp_bias=True))
+@settings(max_examples=10000)
```

Comment:
@Giuseppe5, any thoughts on a good value here? Maybe `1000` is a good compromise between confidence and test time...

Reply:
Let's do 1K